pkg/ebpf/bytecode/kernel_headers.go

```diff
@@ -0,0 +1,100 @@
+// +build linux_bpf
+
+package bytecode
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+var baseDirGlobs = []string{
+	"/usr/src/kernels/*",
+	"/usr/src/linux-*",
```

Comment:
Using a glob here is dangerous. We could match against a different version of headers versus the kernel, if they had them installed. You can get the equivalent of `uname -r` using this:
```
import (
    "golang.org/x/sys/unix"
    "strings"
)

func kernelTarget() (string, error) {
	var uname unix.Utsname
	err := unix.Uname(&uname)
	if err != nil {
		return "", err
	}
	return strings.Trim(string(uname.Release[:]), "\x00"), nil
}
```